Makefile

```diff
@@ -10,6 +10,7 @@ default:
 	@echo "Welcome to the buildhub\n"
 	@echo "The list of commands for local development:\n"
 	@echo "  build            Builds the docker images for the docker-compose setup"
+	@echo "  setup            Initializes and sets up Postgres and Elasticsearch services"
```

Comment:
Having a separate `setup` rule makes it easier to throw away any state and reinitialize services.